### ISSUE TITLE
Shallow copies of Tensor objects made in postprocess causing invalid bounding box co-ordinates

### ISSUE THREAD

**Issue by blackhole077** [haotian-liu/yolact_edge#195]
**Describe the bug**
In the postprocess function, found within output_utils.py there are assignments to the following variables.
```
    classes = dets['class']
    boxes   = dets['box']
    scores  = dets['score']
    masks   = dets['mask']
```

These variables are all of class torch.Tensor, and are modified using functions such as F.interpolate and sanitize_coordinates. However, the original Tensors (i.e., dets['box'], etc.) are also affected by these changes, contrary to the expectation set by assigning them to variables. This means that, when running the postprocess function twice per the evalimage function (where it occurs in prep_display and then later in evalimage itself), the absolute co-ordinates of the bounding box are de-normalized twice leading to an invalid output.

Note that, due to the code structure, the annotations (i.e., bounding box, mask and text locations) on the image(s) processed are placed correctly, meaning you need to output the detections in a JSON file to observe the bug.

**To Reproduce**

Using a trained model, the bug should be replicated using the following command:
python eval.py --disable_tensorrt --trained_model=./weights/weight_file_to_use.pth --score_threshold=0.3 --top_k=100 --image=/path/to/image/to/test/image_name.png --output_coco_json

I personally used a custom configuration and trained a model for this process, though I believe that the bug should be present regardless of whether a custom model or a default model is used.

**Expected behavior**
The outputted JSON file should contain absolute co-ordinates that are within the range [0, W] or [0, H] where W and H represent the width and height of the original input image(s).

**Full logs**
Below is an example of evaluating one image (containing a single detection for simplicity) and how it is handled in evalimage
```
[07/14 11:33:19 yolact.eval]: Loading model...
[07/14 11:33:23 yolact.eval]: Model loaded.

[07/14 11:33:26 yolact.eval]: Entering postprocess function from prep_display

[07/14 11:33:26 yolact.output_utils]: Unsanitized X1 and X2: tensor([0.5401])	tensor([0.5593])
[07/14 11:33:26 yolact.output_utils]: Unsanitized Y1 and Y2: tensor([0.5462])	tensor([0.5704])

[07/14 11:33:26 yolact.output_utils]: Sanitized X1 and X2: tensor([297.0758])	tensor([307.5928])
[07/14 11:33:26 yolact.output_utils]: Sanitized Y1 and Y2: tensor([300.3866])	tensor([313.7122])

[07/14 11:33:26 yolact.eval]: Entering postprocess from evalimage
[07/14 11:33:26 yolact.output_utils]: Unsanitized X1 and X2: tensor([297.0758])	tensor([307.5928])
[07/14 11:33:26 yolact.output_utils]: Unsanitized Y1 and Y2: tensor([300.3866])	tensor([313.7122])

[07/14 11:33:26 yolact.output_utils]: Sanitized X1 and X2: tensor([163391.6719])	tensor([550.])
[07/14 11:33:26 yolact.output_utils]: Sanitized Y1 and Y2: tensor([165212.6406])	tensor([550.])
```

**Environment:**
 - OS: PopOS version 22.04 LTS
 - GPU: Nvidia GeForce RTX 3090 * 3
 - CUDA Version 11.03 (Checked using torch._C._cuda_getCompiledVersion())

**Additional context**
This problem can be hotfixed by using copy.deepcopy() on the variables that are modified within the function and modifying the code within postprocess to look like the following:
```
    import copy
    classes = dets['class']
    boxes   = copy.deepcopy(dets['box'])
    scores  = dets['score']
    masks   = copy.deepcopy(dets['mask'])
```

The boxes variable definitely needs this, whereas it is unclear if the others need this treatment or not.
